### PR TITLE
Migrate to traefik v2.4.2

### PR DESCRIPTION
**Pull request by Vboivin**
Changing image version for issue https://github.com/ClubCedille/cloud-sre/issues/136

### CODE REVIEW

File: resources/deployment.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccountName: traefik
       terminationGracePeriodSeconds: 60
       containers:
-      - image: traefik:1.7.22
+      - image: traefik:2.4.2
```
Comment (cguertin14) [inline resources/deployment.yaml:19]: ce serait possible de simplement mettre `traefik` ici (enlever le tag) et de spécifier le tag dans le fichier `kustomization.yml` ?

**Review by cguertin14 — CHANGES_REQUESTED** [review]

File: resources/deployment.yaml (hunk 1 above)
Comment (cguertin14) [reply resources/deployment.yaml:19]: voir: https://stackoverflow.com/a/60333880

**Comment by Vboivin** [comment]
After a quick discussion around a coffee (virtual ofc because covid), Charles and I decided to stay with 1.7.latest for Traefik. Traefik 2.x defines a lot of CRDs and we don't want to lock ourselves to Traefik. We will also solve the traefik port issue by assigning the elasticity in the HPA from "2 to 8" to "1 to 2"

**Comment by Vboivin** [comment]
Closing this PR, going to create a new one reflecting our new changes.

**Pull request closed without merging**